enterprise_access/apps/subsidy_access_policy/tests/test_models.py

```diff
@@ -952,12 +952,31 @@ def test_redeem(
                 state=assignment_starting_state,
             )
 
+        # Do the redemption
         if redeem_raises:
             with self.assertRaises(redeem_raises):
                 self.active_policy.redeem(self.lms_user_id, self.course_run_key, [])
         else:
             self.active_policy.redeem(self.lms_user_id, self.course_run_key, [])
 
+        # Assert that we call the subsidy client's `create_subsidy_transaction` method
+        # with the expected payload, but only for test conditions where redeem() doesn't
+        # fail before getting to that point.
+        if fail_subsidy_create_transaction or not redeem_raises:
+            expected_redeem_payload = {
+                'subsidy_uuid': str(self.active_policy.subsidy_uuid),
+                'lms_user_id': self.lms_user_id,
+                'content_key': self.course_run_key,
+                'subsidy_access_policy_uuid': str(self.active_policy.uuid),
+                'metadata': None,
+                'idempotency_key': ANY,
```

Comment:
we don't particularly care about what idempotency key is generated in the context of this test, that functionality is tested elsewhere